ex1.py

```diff
@@ -0,0 +1,17 @@
+from collections import defaultdict
+
+
+def group_by(function, iterable):
+    #  this function groups values by the returned value
```

Comment:
Each sentence in the documentation starts with a capital letter and ends with a .